aerospike_discovery.py

```diff
@@ -488,54 +575,172 @@ def _authenticate(self, user, password, password_field_id):
             sys.exit(STATE_UNKNOWN)
     except Exception as e:
         print("Failed to authenticate connection to the Aerospike cluster at %s:%s"%(args.host,args.port))
-        print e
+        print(e)
+        sys.exit(STATE_UNKNOWN)
+
+asinfo_cmd = "statistics"
+
+# Find server version to find correct asinfo commands.
+req = 'build'
+try:
+    version = client.info(req).split('.')
+except Exception as e:
+        print("Failed to execute asinfo command %s on the Aerospike cluster at %s:%s"%(req, args.host, args.port))
+        print(e)
         sys.exit(STATE_UNKNOWN)
 
+use_latencies_cmd = True if int(version[0]) >= 5 and int(version[1]) >= 1 else False
```

Comment:
`True if` and `False` not needed, the comparison will return a bool. Also, I think this will brake with server versions like `60`

```
>>> v = '60'
>>> int(v[0]) >= 5 and int(v[1]) >= 1
False
```

Reply:
Ha!  What a silly mistake.

Reply:
I think the changed version will still break with server versions like 6.0,  `60`